large-message-connect/src/main/java/com/bakdata/kafka/LargeMessageConverterConfig.java

```diff
@@ -77,15 +75,6 @@ private static ConfigDef configDef() {
                         CONVERTER_CLASS_DOC);
     }
 
-    @Override
-    protected HeaderDeserializationStrategy getHeaderDeserializationStrategy() {
-        // We cannot remove headers when deserializing because we need to retain them in case a dead letter is
-        // created from the respective record. This is done without calling the converter.
-        // Because there are no producer records created from a record with sink connectors, this cannot lead to
-        // corrupted headers
-        return RETAIN;
```

Comment:
If we are not retaining headers anymore what happens to dead letter records then?

Reply:
We retain them because we do not modify headers. We just drop them in streams

Reply:
(Tests for connect did not need any change, so that proves it is working)